app/views/streams/main_stream.html.haml

```diff
@@ -22,42 +22,42 @@
 
 %div.container-fluid
   .row
-    .left-navbar-fixed-sub-background.col-lg-1.visible-lg
-    .left-navbar-fixed-background.col-lg-offset-1.col-lg-2.col-md-3.hidden-sm.hidden-xs
-    .left-navbar.col-lg-offset-1.col-lg-2.col-md-3
-      #home_user_badge
-        = owner_image_link
-        %h4
-          = link_to current_user.first_name, local_or_remote_person_path(current_user.person)
-
-      %ul#stream_selection
-        %li{data: {stream: "stream"}}
-          = link_to t("streams.multi.title"), stream_path, rel: "backbone", class: "hoverable"
-        %li{data: {stream: "activity"}}
-          = link_to t("streams.activity.title"), activity_stream_path, rel: "backbone", class: "hoverable"
-        %li{data: {stream: "mentions"}}
-          = link_to t("streams.mentions.title"), mentioned_stream_path, rel: "backbone", class: "hoverable"
-        %li.all_aspects
-          = render "aspects/aspect_listings", stream: @stream
-        %li.followed-tags-sidebar
-          = render "tags/followed_tags_listings"
-        %li{data: {stream: "public"}}
-          = link_to t("streams.public.title"), public_stream_path, rel: "backbone", class: "hoverable"
-
-    .col-md-6
-      #aspect_stream_container.stream_container
+    .col-lg-1.hidden-xs.hidden-sm.hidden-md.visible-lg
+    .col-xs-12.col-lg-10.stream-wrapper.col-md-11-with-chat-shown
+      .left-navbar.col-md-3
+        #home_user_badge
+          = owner_image_link
+          %h4.hidden-xs
+            = link_to current_user.first_name, local_or_remote_person_path(current_user.person)
+
+        %ul#stream_selection
+          %li{data: {stream: "stream"}}
+            = link_to t("streams.multi.title"), stream_path, rel: "backbone", class: "hoverable"
+          %li{data: {stream: "activity"}}
+            = link_to t("streams.activity.title"), activity_stream_path, rel: "backbone", class: "hoverable"
+          %li{data: {stream: "mentions"}}
+            = link_to t("streams.mentions.title"), mentioned_stream_path, rel: "backbone", class: "hoverable"
+          %li.all_aspects
+            = render "aspects/aspect_listings", stream: @stream
+          %li.followed-tags-sidebar
+            = render "tags/followed_tags_listings"
+          %li{data: {stream: "public"}}
+            = link_to t("streams.public.title"), public_stream_path, rel: "backbone", class: "hoverable"
+
+      #aspect_stream_container.stream_container.col-md-7.col-lg-6.col-md-9-with-chat-shown
```

Comment:
Classes should be listed before IDs (.stream_container should precede #aspect_stream_container)